tests/OrbitStubClientTest.php

```diff
@@ -29,11 +29,25 @@ public function testGetContentWithPresentButUnusedConstructor()
         );
 
         $expectedOrbit = new Orbit(
-            '<orbit-head/>',
+            '<orbit-head><orbit-template-params data-values="[]"/></orbit-head>',
             '<orbit-bodyfirst/>',
             '<orbit-bodylast/>'
         );
 
         $this->assertEquals($expectedOrbit, $orbitClient->getContent());
     }
+
+    public function testGetContentWithTemplateParameters()
+    {
+        $orbitClient = new OrbitStubClient();
+
+        $expectedOrbit = new Orbit(
+            '<orbit-head><orbit-template-params data-values="'
+                    . htmlentities(json_encode(['testKey' => 'testValue'])) .'"/></orbit-head>',
```

Comment:
Would it be better to be explicit with our expected output ` <orbit-head><orbit-template-params data-values="[&quot;testKey&quot;=>&quot;testValue&quot;]"/></orbit-head>`?

Then we can be sure we're getting the right output. At the moment you're testing `htmlentities(json_encode(['testKey' => 'testValue']))` is the same output as `htmlentities(json_encode(['testKey' => 'testValue']))` which doesn't seem overly helpful :)